### DISCUSSION TITLE
can not use include with select when fetching data

### DISCUSSION THREAD

**Discussion by BishoyNarouz** [prisma/prisma#6307] [Q&A]
1) i can not use both select and include together in get data
2) in upsert command i can not use select (only i can use include) how can i use select in upsert?

**Comment by ryands17** [comment]
@BishoyNarouz 👋 

1. `select` cannot be used with `include` but you can achieve the same functionality with both
2. `upsert` supports both `select` and `include`.

Could you share the schema and command you're running where `select` is not supported?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
